Controllers/HomeController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+	public class HomeController : Controller
+	{
+		[Route("", Name ="Home", Order = 0)]
+		[Route("app", Name = "app", Order = 0)]

```

Comment:
I ask because at the moment, these names are not meaningful to me and add complexity to the code because when I read them, I try to figure out how they are used and can't find anything.